src/com/google/README.md

```diff
@@ -0,0 +1,7 @@
+# Protocol Buffer Specifications
```

Comment:
A minor suggestion @ghaskins - perhaps add a statement that files under src/com/google/protobuf are generated from these proto files ?

Reply:
I like it, will do.  Thanks Murali